### DISCUSSION TITLE
Merging `cva` objects

### DISCUSSION THREAD

**Discussion by charmbyte** [chakra-ui/panda#976] [Q&A]
Is it possible to merge two `cva` objects to keep things DRY?

For example, say I have two typography components - Heading and Text - and I want to define Heading variants using `cva` that inherits all of the variants from the Text component.

Coming from using StitchesJS,  the way to have done this would be to supply the `headingStyles` call with the first argument being `textStyles` and the second argument would be the variants and styles specific to the heading component. So, for example, I have a function `textStyles = cva(...)` and I want a second function that looks something like `headingStyles = cva(textStyles, ...)`.

Is there an equivalent to this in Panda?

Thanks!

**Comment by anubra266** [comment]
No there isn't anything like this @charmbyte IMO heading should be a variant in textStyles
